cmd/kots/cli/velero.go

```diff
@@ -44,6 +48,8 @@ func VeleroCmd() *cobra.Command {
 	cmd.AddCommand(VeleroConfigureNFSCmd())
 	cmd.AddCommand(VeleroConfigureHostPathCmd())
 	cmd.AddCommand(VeleroPrintFileSystemInstructionsCmd())
+	// TODO (dans)
+	// cmd.AddCommand(VeleroMigrationFileSystemCmd())
```

Comment:
do we still need such a command?

Reply:
Yes, this will be what the addon calls to execute the migration since it doesn't run `upgrade`.